packages/contracts-bedrock/scripts/deploy/DeployConfig.s.sol

```diff
@@ -59,6 +59,8 @@ contract DeployConfig is Script {
     uint256 public l2GenesisBlockGasLimit;
     uint32 public basefeeScalar;
     uint32 public blobbasefeeScalar;
+    uint32 public operatorFeeScalar;
+    uint64 public operatorFeeConstant;
```

Comment:
This file doesnt set the recipient so the recipient should be deleted everywhere in config files